lowcar/devices/PolarBear/PolarBear.cpp

```diff
@@ -1,96 +1,77 @@
 #include "PolarBear.h"
 
+// enumerate the parameters
 typedef enum {
-	DUTY_CYCLE = 0,
-	MOTOR_CURRENT = 1,
-	DEADBAND = 2
+    DUTY_CYCLE = 0,
+    MOTOR_CURRENT = 1,
+    DEADBAND = 2
 } param;
 
-PolarBear::PolarBear () : Device (DeviceType::POLAR_BEAR, 2) {
-	this->duty_cycle = 0;
-	this->deadband = 0.05;
-
-	this->dpwm_dt = 255 / 200000;
+PolarBear::PolarBea () : Device(DeviceType::POLAR_BEAR, 2) {
```

Comment:
typo: PolarBear

Reply:
o god how did I not see this one / how did it compile??